### ISSUE TITLE
[Work Item]

### ISSUE THREAD

**Issue by C0ubv9** [microsoft/CSS-Exchange#146]
**Describe the work**
Script discrepancy

**Additional Context**
Found some discrepancy between this post and the script currently in github. 
https://www.microsoft.com/security/blog/2021/03/02/hafnium-targeting-exchange-servers/
On the post above, for CVE-2021-26855, it contains - 
Where-Object {  $_.AuthenticatedUser -eq '' -and $_.AnchorMailbox -like 'ServerInfo~*/*' }

But current version on github it doesn't contain ($_.AuthenticatedUser -eq '') condition.

Would you please explain why the current script did not include the authenticateduser as one of conditions or should it be included? Thanks.

**Comment by bill-long** [comment]
The thinking with `-AuthenticatedUser -eq ''` is that we don't need to look at requests that are authenticated. From what I see, the script has never had that parameter, even back to its initial version. As far as I can tell, we're not actually seeing any authenticated requests that have a ServerInfo hit.

So the short answer is that it makes no practical difference based on the data we're seeing right now. However, we should probably add that parameter just for consistency with the blog post.

**Comment by C0ubv9** [comment]
Thanks for a lot for the detailed explanations. We are currently re-run modified script with condition added and will keep you posted if we see different results.

**Comment by C0ubv9** [comment]
Yes rerunning returned same results. Again thanks for the quick response.
